spark/src/main/scala/io/delta/tables/execution/DeltaTableOperations.scala

```diff
@@ -112,6 +113,41 @@ trait DeltaTableOperations extends AnalysisHelper { self: DeltaTable =>
     toDataset(sparkSession, restore)
   }
 
+  protected def executeClone(
+    table: DeltaTableV2,
```

Comment:
Let's use 4 space indentation for method declarations (basically other functions here) like https://github.com/databricks/scala-style-guide

Reply:
Updated